test/fixtures/non-existing-elm-in-sequence.json

```diff
@@ -0,0 +1,66 @@
+{
+  "element": "parseResult",
+  "meta": {},
+  "attributes": {},
+  "content": [
+    {
+      "element": "annotation",
+      "meta": {
+        "classes": [
+          "error"
+        ],
+        "links": [
+          {
+            "element": "link",
+            "meta": {},
+            "attributes": {
+              "relation": "origin",
+              "href": "http://docs.apiary.io/validations/swagger#swagger-validation"
+            },
+            "content": []
```

Comment:
Oh wait, nvm. This is link. What I meant to say was there is no source map of this annotation.